### PR TITLE
AIX: Update manual setup instructions

### PR DESCRIPTION
**Pull request by gibfahn**
doc: update AIX manual install instructions
- Convert txt -> md
- Remove obsolete sections
- Document curl LIBPATH fix
- Document pip install instructions

Refs: #551
Refs: #533

To view the markdown (if the diff doesn't work): https://github.com/gibfahn/build/blob/aix-pip/setup/aix61/manualBootstrap.md

cc/ @mhdawson @sxa555

### CODE REVIEW

File: setup/aix61/manualBootstrap.md [hunk 1]
```diff
@@ -85,60 +122,59 @@ ln -s /opt/freeware/bin/ccache/ccache /opt/freeware/bin/ccache/g++
 ln -s /opt/freeware/bin/ccache/ccache /opt/freeware/bin/ccache/gcc
 cd ../..
 rm -rf tmp
+```
+
+## Install svn
 
-5) install svn, only needed for v8 testing
+*N.B. Only needed for v8 testing*
```
Comment (gibfahn) [inline setup/aix61/manualBootstrap.md]: @mhdawson do we still need to install svn? I thought we'd moved to git for the v8 testing. If so I'll remove this as well.

Comment (mhdawson) [inline setup/aix61/manualBootstrap.md]: I think you're right, we had to switch over a while back.

Comment (jbergstroem) [inline setup/aix61/manualBootstrap.md]: Does this mean we can drop svn from our global dependencies for all builders?

Comment (mhdawson) [inline setup/aix61/manualBootstrap.md]: Likely yes.  We'd want to remove on one set of machines fist to validate but if we are right we don't need it anymore.

**Pull request merged**